src/proteus/proteus.py

```diff
@@ -229,12 +231,6 @@ def start(self, *, resume: bool = False):
             self.loops["init"] = self.loops["init_loops"] + 1
         log.info(" ")
 
-        # Create lockfile for keeping simulation running
-        self.lockfile = CreateLockFile(self.directories["output"])
-
-        # Download basic data
-        download_sufficient_data(self.config)
```

Comment:
Same here?

Reply:
This is still included in the code and is very much necessary. I had to move it further up in the file because it should be called before the initial structure calculation, because we need to make sure that the lookup tables are downloaded before trying to run Aragog.